### PR TITLE
TRUNK-5905 : Replaced xml mapping with annotations for ConceptMapType

### PR DESCRIPTION
**Pull request by LuGO0**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5905

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by LuGO0** [comment]
I did similar changes here for ConceptMapType and the PR has no build Issues but the PR for FieldType is creating Issues, will look into the older PR again, please review this PR @tendomart @dkayiwa .

**Comment by LuGO0** [comment]
@HerbertYiga @dkayiwa can we merge this ?

File: api/src/main/java/org/openmrs/ConceptMapType.java [hunk 1]
```diff
@@ -11,18 +11,31 @@
...
  * @since 1.9
  */
+@Entity
+@Table(name = "concept_map_type")
 public class ConceptMapType extends BaseChangeableOpenmrsMetadata {
-	
+
 	private static final long serialVersionUID = 1L;
-	
+
```
Comment (HerbertYiga) [inline api/src/main/java/org/openmrs/ConceptMapType.java:32]: remove this space

File: api/src/main/java/org/openmrs/ConceptMapType.java [hunk 2]
```diff
@@ -11,18 +11,31 @@
...
 /**
  * ConceptMapType are used to define relationships between concepts and concept reference terms e.g
  * IS_A or SAME_AS, BROADER_THAN
  *
  * @since 1.9
  */
+@Entity
+@Table(name = "concept_map_type")
 public class ConceptMapType extends BaseChangeableOpenmrsMetadata {
```
Comment (HerbertYiga) [inline api/src/main/java/org/openmrs/ConceptMapType.java:29]: have you cross checked that all the attributes that you never included in this file do exist under the extended class

Comment (LuGO0) [reply api/src/main/java/org/openmrs/ConceptMapType.java:29]: yes @HerbertYiga

File: api/src/main/java/org/openmrs/ConceptMapType.java (hunk 1 above)
Comment (LuGO0) [reply api/src/main/java/org/openmrs/ConceptMapType.java:32]: There is no extra space added @HerbertYiga on its some glitch with the version control which shows that line as deleted and then added again on its own. not sure how to remove that?

**Comment by LuGO0** [comment]
@HerbertYiga should we close the Issue after merging this?

**Pull request merged**